config/vxlan.py

```diff
@@ -23,7 +24,9 @@ def add_vxlan(db, vxlan_name, src_ip):
 
     if ADHOC_VALIDATION:
         if not clicommon.is_ipaddress(src_ip):
-            ctx.fail("{} invalid src ip address".format(src_ip))  
+            ctx.fail("{} invalid src ip address".format(src_ip))
+        if len(vxlan_name) >= IFNAMSIZ:
```

Comment:
@arfeigin is this a typo? What should be `>=` or `>`?

Reply:
as above